Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM postgres:12.2
```

Comment:
`postgres:12.3` is out. Shall we upgrade?

Reply:
I guess we bump postgres version in major release, is this not correct ?

Reply:
Not neccessarily. We usually try to bump PG version once it becomes available.